server/utils/premises/index.test.ts

```diff
@@ -240,6 +241,50 @@ describe('premisesUtils', () => {
     })
   })
 
+  describe('groupCas1SummaryPremisesSelectOptions', () => {
+    const area1Premises = cas1PremisesSummaryFactory.buildList(2, { apArea: { id: 'a1', name: 'Area 1' } })
+    const area2Premises = cas1PremisesSummaryFactory.buildList(2, { apArea: { id: 'a2', name: 'Area 2' } })
+    const premises = [...area1Premises, ...area2Premises]
+
+    it('should group premises by region', () => {
+      expect(groupCas1SummaryPremisesSelectOptions(premises, { premisesId: area1Premises[1].id })).toEqual([
+        {
+          items: [
+            { selected: false, text: area1Premises[0].name, value: area1Premises[0].id },
+            { selected: true, text: area1Premises[1].name, value: area1Premises[1].id },
+          ],
+          label: 'Area 1',
+        },
+        {
+          items: [
+            { selected: false, text: area2Premises[0].name, value: area2Premises[0].id },
+            { selected: false, text: area2Premises[1].name, value: area2Premises[1].id },
+          ],
+          label: 'Area 2',
+        },
+      ])
+    })
+
+    it('should support a field name', () => {
```

Comment:
What does the field name do? Can this be made a bit more explicit, i.e. 'marks the correct option as selected'

Reply:
It was a copy'n'paste from the original so yes, it's not very precise.